hack/lib/ui.bash

```diff
@@ -20,6 +20,29 @@ readonly COLOR_CYAN='\e[0;36m'
 readonly COLOR_LIGHT_RED='\e[1;31m'
 readonly COLOR_LIGHT_YELLOW='\e[1;33m'
 
+function debugging.setup {
```

Comment:
Could you move the unrelated changes into their own PR please?

Reply:
Cool. Will do.